libexec/dnf-utils.in

```diff
@@ -95,11 +92,11 @@ if command == 'package-cleanup':
         MAPPING[command] = ['remove']
     elif '--oldkernels' in args:
         args[args.index('--oldkernels')] = '--oldinstallonly'
+        args.append('--setopt=installonlypkgs=kernel,kernel-devel')
```

Comment:
Oh, you're right, thanks.

Any idea what to do with the `--oldkernels` option then?  It seems we only have two options:
1) remove it completely (and maybe print a message when attempting to use it)
2) keep it mapped to `--oldinstallonly` (which is a lie, since not only kernels are removed)

Reply:
I'm going to remove it for now (it will not be the only option that's no longer available, so not a big deal I guess). We can always resolve this later if really needed.